module/actor.js

```diff
@@ -1014,6 +1025,24 @@ class DCCActor extends Actor {
       })
     }
 
+    // Add Strength or Agility modifier to attack rolls
+    let modifier;
+    let modifierLabel;
+    if ((this.system.class.className) && (game.settings.get('dcc', 'automateCombatModifier'))) {
+      if (weapon.system.melee) {
+          modifier = " + " + this.system.abilities.str.mod;
+          modifierLabel = "DCC.AbilityStr";
+      } else {
+          modifier = " + " + this.system.abilities.agl.mod;
```

Comment:
Is there a particular reason we need to append a plus on to this and force it to a string? What if the modifier is negative? Similarly for the other modifiers being added in this change.

Reply:
It s not needed - removed.